src/SFA.DAS.Admin.Roatp.Web/Controllers/OrganisationTypeUpdateController.cs

```diff
@@ -23,14 +23,14 @@ public async Task<IActionResult> Index(int ukprn, CancellationToken cancellation
 
         foreach (var organisationType in organisationTypesResponse.OrganisationTypes)
         {
-            organisationTypes.Add(new OrganisationTypeModel { Id = organisationType.Id, Description = organisationType.Description, IsSelected = organisationType.Id == organisationResponse.OrganisationTypeId });
+            organisationTypes.Add(new OrganisationTypeModel { Id = organisationType.Id, Description = organisationType.Description, IsSelected = organisationType.Id == organisationResponse.Content!.OrganisationTypeId });
```

Comment:
create a variable for organisationResponse.Content! and use that in the controller instead of all the references